redlines/redlines.py

```
@@ -1,62 +1,155 @@
+                "\n"
+                "Cause: The source property was accessed before being set.\n"
+                "\n"
+                "To fix: Initialize Redlines with source text or set it explicitly:\n"
+                "  # Option 1: Initialize with both texts\n"
+                "  redlines_obj = Redlines(source='original', test='modified')\n"
+                "\n"
+                "  # Option 2: Set source property after creation\n"
+                "  redlines_obj = Redlines()\n"
+                "  redlines_obj.source = 'original'\n"
+                "  redlines_obj.test = 'modified'\n"
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)